Software/Python/ir_remote_control/gobox_ir_receiver_libs/ir_debug_logger.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python 
```

Comment:
If we ever want to run any of these scripts as standalone executables, the shebang helps. http://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take